examples/DDPG/replay_memory.py

```diff
@@ -0,0 +1,43 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+
+class ReplayMemory(object):
+    def __init__(self, size, obs_dim, act_dim):
+        self.size = size
+        self.obs_dim = obs_dim
+        self.act_dim = act_dim
+        self.memory = np.zeros((size, obs_dim * 2 + act_dim + 2),
+                               dtype=np.float32)
+        self.pointer = 0
+
+    def sample_batch(self, batch_size):
```

Comment:
This implementation is ok, but I think it's easy to mistake the indices here.
Btw, why don't you build the RPM using a similar structure like that in DQN? 
Keeping the consistency between different RPMs could make our framework more readable for users.